### DISCUSSION TITLE
Flight plan is  simplified: airways implemented ?

### DISCUSSION THREAD

**Discussion by JLD95** [flybywiresim/aircraft#2340] [Q&A]
Hi
With SimBrief, I made a FP between LOWW and ESMS:
LANUX6D LANUX L858 HDO M725 KOGIM M44 ARGAD Z400 BAKLI BAKLI1S
Under FS2020, the Airways L858 seems ignored by FMS: A320 made a straight line between LANUX and BUSIR rather a poyline between LANUX, OKF, VLM, RISUK, ULNIP, HDO and BUSIR.

**Comment by MapleNAV-TCASFAULT** [comment]
This may be a nav data issue with Asobo, with some airways being incomplete.

**Comment by sai-barath** [comment]
Are you sure you’re on the latest master?

**Comment by JLD95** [comment]
No  i have master 2020-12-05T18, the latest master has corrected this phenomenon?Le 10 déc. 2020 19:56, Username23-23 <notifications@github.com> a écrit :

Are you sure you’re on the latest master?

**Comment by ImenesFBW** [comment]
Did you import the simbrief plan via the pre-game menu, or did you enter it via the mcdu? The first option will not work correctly.

**Comment by JLD95** [comment]
I use pre-game menu. For load FPN on MCDU, the same pln format is used or a specific file format is necessary?Le 10 déc. 2020 21:05, Imenes <notifications@github.com> a écrit :
Did you import the simbrief plan via the pre-game menu, or did you enter it via the mcdu? The first option will not work correctly.

**Reply by ImenesFBW** [reply]
The pre-game menu import is not yet fully supported, to get the route correct you need to use the MCDU FPLN page. This will be made easier with the simbrief integration that is coming soon.
